PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h

```diff
@@ -2283,26 +2708,242 @@ Bool_t MaxNumberOfEvents()
 
 //============================================================
 
+Double_t CalculateCustomNestedLoop(TArrayI* harmonics)
+{
+  // For the specified harmonics, get the correlation from nested loops.
+  // Order of correlator is the number of harmonics, i.e. the number of elements in an array.
+
+  // a) Determine the order of correlator;
+  // b) Custom nested loop;
+  // c) Return value.
+
+  if (tc.fVerbose) {
+    LOGF(info, "\033[1;32m%s\033[0m", __PRETTY_FUNCTION__);
+  }
+
+  if (!harmonics) {
+    cout << __LINE__ << endl;
+    exit(1);
```

Comment:
Please, for the next iteration substitute `cout` by `LOGF(fatal, ...)` and `exit` by `return`

Reply:
Yes, I overlook a few cases. Noted!